index.js

```diff
@@ -136,7 +144,7 @@ const validateResponse = (req, res, next) => {
 
     const responseSchema = resolveResponseModelSchema(req, res);
     if (!responseSchema) {
-      debug('Response validation skipped: no matching response schema');
+      debug(`Response validation skipped: no matching response schema for ${req.method} ${req.originalUrl}`);
```

Comment:
The intent behind these changes is to make it easier to see which requests are not covered by a given schema.